src/query/execute_schema.c

```diff
@@ -2959,6 +2961,11 @@ do_create_index (PARSER_CONTEXT * parser, const PT_NODE * statement)
 
   index_name = statement->info.index.index_name ? statement->info.index.index_name->info.name.original : NULL;
 
+  if (statement->info.index.index_status == SM_ONLINE_INDEX_BUILDING_IN_PROGRESS)
+    {
+      ib_thread_count = statement->info.index.ib_threads;
+    }
+
```

Comment:
This was done so that we won't pass the `ib_thread_count` through all the schema functions as a parameter.